Content.Shared/Cuffs/SharedCuffableSystem.cs

```diff
@@ -484,7 +484,6 @@ public bool TryCuffing(EntityUid user, EntityUid target, EntityUid handcuff, Han
                 BreakOnWeightlessMove = false,
                 BreakOnDamage = true,
                 NeedHand = true,
-                DistanceThreshold = 0.3f
```

Comment:
This is one tile away this is insane distance for cuffing

Reply:
alright, how about one

Reply:
It's already breakonmove why does this need to be non-default.

Reply:
1's fine, general interaction is pretty long and it feels weird cuffing someone from so far away